app/views/state_file/questions/nj_review/edit.html.erb

```diff
@@ -16,171 +16,195 @@
+        <% end %>
+      </div>
+    </section>
+  <% end %>
+
+  <% if current_intake.filing_status_qw? %>
+    <section id="year-of-death" class="white-group">
+      <div class="spacing-below-5">
+        <h2 class="text--body text--bold spacing-below-5"><%=t(".year_of_death") %></h2>
+        <p><%=current_intake.spouse_death_year %></p>
 
-  <div id="municipality" class="white-group">
+        <%= link_to StateFile::Questions::NjYearOfDeathController.to_path_helper(return_to_review: "y"), class: "button--small" do %>
+          <%= t(".review_and_edit") %>
+          <span class="sr-only"><%= t(".year_of_death") %></span>
+        <% end %>
+      </div>
+    </section>
+  <% end %>
+
+  <section id="county-municipality" class="white-group">
     <div class="spacing-below-5">
-      <h2 class="text--body text--bold spacing-below-5"><%=t(".municipality") %></h2>
-      <p><%=current_intake.municipality_name %></p>
-      <%= link_to StateFile::Questions::NjMunicipalityController.to_path_helper(return_to_review: "y"), class: "button--small" do %>
-        <%= t("general.edit") %>
-        <span class="sr-only"><%= t(".municipality") %></span>
+      <h2 class="text--body text--bold spacing-below-5"><%=t(".county_municipality", filing_year: current_tax_year) %></h2>
+      <p><%=current_intake.county %>, <%=current_intake.municipality_name %></p>
+
+      <%= link_to StateFile::Questions::NjCountyController.to_path_helper(return_to_review: "y"), class: "button--small" do %>
+        <%= t(".review_and_edit") %>
+        <span class="sr-only"><%= t(".county_municipality") %></span>
       <% end %>
     </div>
-  </div>
+  </section>
 
   <% if !current_intake.primary_disabled_unfilled? || !current_intake.spouse_disabled_unfilled? %>
-    <div id="disabled_exemption" class="white-group">
+    <section id="disabled_exemption" class="white-group">
       <div class="spacing-below-5">
-        <% if !current_intake.primary_disabled_unfilled? %>
-          <h2 class="text--body text--bold spacing-below-5"><%=t(".primary_disabled") %></h2>
-          <p><%= current_intake.primary_disabled %></p>
-        <% end %>
-        <% if !current_intake.spouse_disabled_unfilled? %>
-          <h2 class="text--body text--bold spacing-below-5"><%=t(".spouse_disabled") %></h2>
-          <p><%= current_intake.spouse_disabled  %></p>
-        <% end %>
+        <% unless current_intake.primary_disabled_unfilled? %>
+          <h2 class="text--body text--bold spacing-below-5"><%= t(".primary_disabled")  %></h2>
+          <p><%= current_intake.primary_disabled_yes? ? t("general.affirmative") : t("general.negative") %></p>
+        <%  end %>
+        <% unless current_intake.spouse_disabled_unfilled? %>
+          <h2 class="text--body text--bold spacing-below-5"><%= t(".spouse_disabled")  %></h2>
+          <p><%= current_intake.spouse_disabled_yes? ? t("general.affirmative") : t("general.negative") %></p>
+        <%  end %>
```

Comment:
[pebble] there seem to be a few of these `<%  end %>` with two spaces throughout the file, some already here and some new